app/views/users/_form.html.erb

```diff
@@ -0,0 +1,34 @@
+<% if false %>
+<form action="/users" method="post" accept-charset="UTF-8">
+     <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>"/>
+     <label for="username" class="form-label">Username : </label>
+     <input name="username" type="text" class="form-control" />
+     <label for="email" class="form-label">Email : </label>
+     <input name="email" type="text" class="form-control" />
+     <label for="password" class="form-label">Password : </label>
+     <input name="password" type="password" class="form-control" />
+     <input type="submit" value="Submit" class="btn btn-primary" />
+   </form>
+
+<% end %>
```

Comment:
[OPTIONAL] Please remove logic from views. You can use helper under app/helpers to put your logic in and render it in views as you need.